index.js

```diff
@@ -177,6 +177,7 @@ function setConfig(configObject) {
     clientId: configObject && configObject.clientId || process.env.IOPIPE_TOKEN || process.env.IOPIPE_CLIENTID || '',
     debug: configObject && configObject.debug || process.env.IOPIPE_DEBUG || false,
     network_timeout: 5000,
+    timeout_millis: 50
```

Comment:
`halt_early_ms`?
`timeout_buffer_ms`?
`minimum_network_timeout`? -- turn it around and say it's the minimum time we allow to send the data?